command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 02:03:13PM -0700, Julian Friedman wrote:
> Yes, the original reason we hesitated to specify the cli was worries
> it'd limit ux choices, the reason we went back on that was realising
> that higher level things - like compliance tools and orchestrators -
> *need* an API in order to actually interact with things and allow
> compliant runtimes to be swapped in and out (which to me is kind of
> the point of a spec)

And again, requiring the runtime to support an API (or one from a set
of choices) does not limit their freedom to also support other,
unspecified or externally-specified APIs as they see fit.  It just
means that the compliance tooling will drive them through the
OCI-specified API(s).